cli/src/commands/branch.rs

```diff
@@ -569,6 +569,17 @@ fn cmd_branch_track(
             names.len()
         )?;
     }
+
+    //show conflicted branches if there are some
+    let list_args = BranchListArgs {
+        all_remotes: true,
+        tracked: true,
+        conflicted: true,
+        names: vec![],
+        revisions: vec![],
+        template: None,
+    };
+    cmd_branch_list(ui, command, &list_args)?;
```

Comment:
You'll probably need to copy the templating part from `cmd_branch_list()` because
* it shouldn't print existing unrelated conflicts.
* it's probably better to print as `ui.status_formatter()` message.

Reply:
oh yes I didn't think about potential existing conflicts before running track